### DISCUSSION TITLE
Translate the book into Chinese

### DISCUSSION THREAD

**Discussion by duke-git** [teivah/100-go-mistakes#78] [Discussions]
Hi @teivah , I am writing this message because of your book 《100 Go Mistakes and How to Avoid Them》. I am Dudaodong, a software engineer from China, who has been engaged in software development for 14 years. Currently working for KingSoft Cloud Technology Co., Ltd. as a web developer. I'm grateful you wrote this excellent book. The content of this book have a great guide for me and my work. I think this book will be a great help for engineers or students who are about to enter the go development world. However, since the book is an original English version, it may be difficult for some Chinese readers to read and understand it. If there is a Chinese translation, more Chinese readers will definitely benefit from it. I don't know if you allow the book to be translated into Chinese then publish it. If so, could I get the translation rights? Looking forward to your reply. Thanks.

**Comment by teivah** [comment]
Hey @duke-git, there's already a Chinese translation: https://phei.com.cn/module/goods/wssd_content.jsp?bookid=64073.

**Reply by duke-git** [reply]
Okay 👍
